### PR TITLE
Next milestone

### PR DESCRIPTION
**Pull request by imran5644**
I create the Components with a basic CSS styling:

Restructure the project by adding Components and a stylesheet folder
Create Display component
Create Button component
Create ButtonPanel component
Use PropTypes to check types of the properties passed to the component
Use default props to add default properties to the Display component
Import CSS files on the components

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const App = () => (
```
Comment (Forison) [inline src/components/App.js:4]: * [x] `App.js` has children components wrapped in React.Fragment short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))
The React convention can make you create nodes that you don't really need sometimes.  With the aid of React fragments no extra node is created, and the code efficiency is improved.
> We are not adding styling yet so we can better appreciate the concept

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
```
Comment (Forison) [inline src/components/ButtonPanel.js]: **[Optional]**
This could also be rendered with a fragment

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Change Required❌

## 👋 @imran5644 

#### Good job on the pull request, it very informing. The required directory and all the required components were created and props validated. Here are a few issues we have to address to get this project approved🤗.

- Check my comment below👇.

> Note: Using next-milestone in my opinion  is not good semantic, you could use milestone-2 etc... It good practice to have a descriptive branch.

#### Happy coding🤗

Addo Forison

**Review by addod19 — COMMENTED** [review]
## Status: Approved :trophy: :balloon: 

Hi @imran5644, congratulations on this milestone :fireworks: The child components have been wrapped with the short syntax of React.Fragment. Awesome :smile: 

I hereby approve this milestone :1st_place_medal: 
Kindly go ahead and merge it into the development branch :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**